composer.json

```diff
@@ -15,7 +15,8 @@
   "require-dev": {
     "phpunit/phpunit": "^8.0",
     "mockery/mockery": "^1.2",
-    "friendsofphp/php-cs-fixer": "^2.14.2"
+    "friendsofphp/php-cs-fixer": "^2.14.2",
+    "ext-xdebug": "*"
```

Comment:
I'm not convinced this is a great idea...

Reply:
It's required to be able to use xdebug_get_headers() for testing purposes. BTW what are you afraid of exactly?

Reply:
I resolve it for now so I can merge it back, but let's talk about it